app/models/concerns/declarable.rb

```diff
@@ -37,6 +37,14 @@ module Declarable
     delegate :code, :short_code, to: :chapter, prefix: true
   end
 
+  def one_or_more_prohibitive_measures?
+    import_measures.present? && import_measures.any?(&:prohibitive?)
```

Comment:
```suggestion
    import_measures.any?(&:prohibitive?)
```
I _think_ `any` will return false when an array of import_measures is empty?